saleor/graphql/channel/types.py

```diff
@@ -75,16 +75,7 @@ def resolve_translation(root: ChannelContext, info, language_code):
         return resolve_translation(root.node, info, language_code)
 
 
-class ChannelContextTypeWithMetadata(ChannelContextDjangoObjectType):
-    """A Graphene type for that uses ChannelContext as root in resolvers.
-
-    Same as ChannelContextType, but for types that implement ObjectWithMetadata
-    interface.
-    """
-
-    class Meta:
-        abstract = True
-
+class MetadataMixin:
     @staticmethod
     def resolve_metadata(root: ChannelContext, info):
```

Comment:
Are you sure that typing is correct here?

Reply:
As discussed offline, this mixin is intended to be used for unwrapping channel context instances.